File: pset1/credit/mario.c
#include<stdio.h>
#include<cs50.h>
void c_ode_(int k);
int main()
{ 
    int m;
    m = get_int("enter height:");
    if (m < 9 &&  m > 0)
    {
        c_ode_(m);
    }
    else
    {
        main();

    }
}
void c_ode_(int k)
{
    int n, j, i ;
    for (i = 1; i <= k; i++)
    {
        for (n = (k - i); n > 0; n--)
        {
            printf(" ");
        }
        for (j = 1; j <= i; j ++)
        {
            printf("#");
        }
        printf("  ");
        for (j = 1; j <= i; j++)
        {
            printf("#");
        }
        printf("\n");
        
    }
}

File: pset6/mario/more/mario.py
from cs50 import get_int


def yo():
    num = get_int("Height: ")
    m(num)
    
    
def m(num):
    if (num <= 0):
        yo()
    elif(num > 8):
        yo()
    else:
        for i in range(1, num + 1):
            for j in range(1, num - i + 1):
                print(" ", end="")
            for k in range(1, i + 1):
                print("#", end="")
            print("  ", end="")
            for k in range(1, i + 1):
                print("#", end="")
            print("")
            
            
yo()